arkouda/pdarrayclass.py

```diff
@@ -6,7 +6,7 @@
 from arkouda.dtypes import structDtypeCodes, NUMBER_FORMAT_STRINGS
 
 __all__ = ["pdarray", "info", "any", "all", "is_sorted", "sum", "prod", "min", "max",
-           "argmin", "argmax", "mean", "var", "std"]
+           "argmin", "argmax", "mean", "var", "std", "mink", "maxk", "argmink", "argmaxk"]
```

Comment:
@reuster986 we have created some new entry points here are you ok with the names we chose? `ak.argmink` to find the indices of the minimum k values in an array and so on.

Reply:
I'm good with these names. Pandas calls these `nlargest` and `nsmallest`, but I think `mink` and `maxk` are easier to remember. I also like the addition of `argmink` and `argmaxk`, which pandas does not have but which are very useful, in my opinion.